src/components/Radio/index.jsx

```diff
@@ -91,6 +82,8 @@ Radio.propTypes = {
 
   /** The radio input `onChange` attribute */
   onChange: PropTypes.func,
+
+  className: PropTypes.string,
```

Comment:
is this something we want to include in props?

Reply:
yeah I'm not sure why I thought this was a good idea 🤣